backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -671,6 +672,123 @@ describe('ContributionResolver', () => {
     })
   })
 
+  describe('denyContribution', () => {
```

Comment:
I suggest to create the needed user here, and after all, clean the database and the token